dluhc-component-library/src/api/api.ts

```diff
@@ -0,0 +1,10 @@
+export async function fetchDataset(coordinates: any[][]) {
+  const points = coordinates[0]
+    .map((point: any[]) => `${point[0]} ${point[1]}`)
+    .join(", ");
+  const polygon = `POLYGON((${points}))`;
```

Comment:
I found some Open Layers formatters that can do this [formatting](https://openlayers.org/en/latest/apidoc/module-ol_format_WKT-WKT.html) too. It would probably mean re-working this function to take `geometry: string` and doing the conversion in the component with something like `formatter.writeFeature(feature)`.

Reply:
will look into this